src/cli.mjs

```diff
@@ -27,6 +27,9 @@ export async function run(argvInput, _container) {
   container = _container;
   const logger = container.resolve("logger");
   const parseYargs = container.resolve("parseYargs");
+  if (process.env.NODE_ENV === "production") {
+    process.removeAllListeners("warning");
```

Comment:
will this have any side effects?

Reply:
only if we accidentally register warning events listeners before this. some more context [here](https://nodejs.org/api/process.html#event-warning).

Reply:
we absolutely can do that in the future - we'd just remove all listeners, then add our listener. listeners are additive, so we can't really override so much as remove, then add.

Reply:
sounds good.